src/test/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/ShardSyncTaskIntegrationTest.java

```diff
@@ -125,7 +126,7 @@ public final void testCall() throws DependencyException, InvalidStateException,
                 InitialPositionInStreamExtended.newInitialPosition(InitialPositionInStream.LATEST),
                 false,
                 false,
-                0L);
+                0L, shardSyncer);
```

Comment:
This looks weird, could you add `shardSyncer` to a new line.

Reply:
Modified